assets/scss/hero.scss

```diff
@@ -54,14 +54,61 @@
     line-height: 1.5;
   }
 
-  figcaption cite {
-    margin-right: 5rem;
-  
-    @media (min-width: $bp-medium) {
-      margin-right: 1rem;
+  figcaption {
```

Comment:
Could you make sure that the hero banner on the homepage for example still looks as before? The following styling updates affect the homepage too.

Reply:
ok fixed this